common/cmake/devices.cmake

```diff
@@ -490,6 +490,7 @@ function(DEFINE_DEVICE)
   #   [CACHE_PLACE_DELAY]
   #   [CACHE_LOOKAHEAD]
   #   [CACHE_ARGS <args>]
+  #   [ROUTE_CHAN_WIDTH]
```

Comment:
Should be `[ROUTE_CHAN_WIDTH <width>]`, yes?

Reply:
Need `[DONT_INSTALL]` in arg summary?

Reply:
I updated the function documentation.